qa/automation/CareerPortal.Tests.Scenarios/2. Extended example/Test/CVforNotOpenPosition.feature

```diff
@@ -0,0 +1,122 @@
+Background: User went to the "Send us your CV" form at the down of the page 
+
+Feature: Send CV for not open job position  
+
+@Happy_pass
+Scenario: Send CV happy pass
+Given User entered valid Name
+And User entered valid e-mail
+And User checked the checkbox
+And User attached CV in .pdf file 10 Mb
+When User press send button
+Then the CV should be received by recruiters
+
+Scenario: Send CV with message happy pass
+Given User entered valid Name
+And User entered valid e-mail
+And User entered 500 symbols in message field
+And User checked the checkbox
+And User attached CV in .docx file 10 Mb
+When User press send button
+Then the CV should be received by recruiters
+
+@negative_tests
+
+Scenario: Send CV with too large message 
+Given User entered valid Name
+And User entered valid e-mail
+And User entered 501 symbols in message field
+And User checked the checkbox
+And User attached CV in .docx file 10 Mb
+When User press send button
+Then CV should be unsend, mistake message about lenght message no more than 500 symbol
```

Comment:
Here and in other Scenarios it is better to describe real results (specify, which field has which text, color). For example, you can write that mistake message should be "The maximum number of characters is 500".

Reply:
I hope, I've corrected this.